source/server/ScriptEngine.h

```diff
@@ -40,237 +40,241 @@ class Sequencer;
 /**
  * This struct holds the information for a script callback.
  */
-struct callback_t {
-    asIScriptObject *obj;  //!< The object instance that will need to be used with the function.
-    asIScriptFunction *func; //!< The function or method pointer that will be called.
+struct callback_t
+{
+   asIScriptObject   *obj;  //!< The object instance that will need to be used with the function.
+   asIScriptFunction *func; //!< The function or method pointer that will be called.
 };
-typedef std::vector<callback_t> callbackList;
+typedef std::vector<callback_t>   callbackList;
 
 class ScriptEngine {

```

Comment:
{ placement of `class` should be consistent with `struct`, and I like it placed on the next line.